lib/ansible/cli/config.py

```diff
@@ -133,14 +115,6 @@ def execute_edit(self):
         '''
         raise AnsibleError("Option not implemented yet")
 
-        # pylint: disable=unreachable
```

Comment:
these are placeholders as hints of what i need to do when implementing the features, see 'pylint hint'